components/tag/src/tag-add/tag-add.ts

```diff
@@ -127,6 +134,11 @@ export class DtTagAdd implements AfterViewInit, OnDestroy {
   constructor(
     private _changeDetectorRef: ChangeDetectorRef,
     private _zone: NgZone,
+    @Optional()
+    @Inject(DT_UI_TEST_CONFIG)
+    private _config?: DtUiTestConfiguration,
+    // @BREAKING-CHANGE: Optional will be removed with version 7.0.0
```

Comment:
Please use a JsDoc comment instead of a normal line comment
```suggestion
    /** @breaking-change: Optional will be removed with version 7.0.0 */
```